Makefile.fossa

```diff
@@ -0,0 +1,32 @@
+REPO_PATH?=docker/buildx
+BUILD_ANALYZER?=dockereng/fossa-analyzer
```

Comment:
oo also heads up, I just made the fossa-analyzer image public so you can change this to: 
`docker/fossa-analyzer`?